source/internal/_clone.js

```diff
@@ -24,15 +24,15 @@ export default function _clone(value, refFrom, refTo, deep) {
     }
     refFrom[idx] = value;
     refTo[idx] = copiedValue;
-    for (var key in value) {
+    for (var key of Object.getOwnPropertyNames(value)) {
```

Comment:
Most of Ramda will work -- believe it or not -- with ES3.  But I'm certain some ES5 has slipped in.

After 1.0, we will skip that and move to more modern JS.

While I like this change, It's not clear whether we should do it now or after 1.0.

Reply:
Thanks for advice, I didn't read the entire documentation(in my language is the last note [pt-br](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyNames)), but just [`Object.create`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create) resolve this issue, I also noticed that this is already in production (#3067), so, can you close this PR and issue(#3070)?

Reply:
Thanks for advice, I didn't read the entire documentation(in my language is the last note [pt-br](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyNames)), but just [`Object.create`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create) resolve this issue, I also noticed that this is already in production (#3067), so, I'm closing this PR